packages/amplify_auth_cognito/ios/Classes/FlutterSignUpRequest.swift

```diff
@@ -36,11 +35,8 @@ struct FlutterSignUpRequest {
     let rawAttributes: Dictionary<String, Any> = options["userAttributes"] as! Dictionary<String, String>
     var formattedAttributes: Array<AuthUserAttribute> = Array()
     for attr in rawAttributes {
-      if (standardAttributes.contains(attr.key)) {
-        formattedAttributes.append(AuthUserAttribute(.unknown(attr.key), value: attr.value as! String))
-      } else {
-        formattedAttributes.append(AuthUserAttribute(.custom(attr.key), value: attr.value as! String))
-      }
+        let stringValue: String = (attr.value is String) ? attr.value as! String : String(attr.value as! Int)
```

Comment:
*Note:* This is outside of the scope of this PR, but we were not handling the scenario where this was an `Int`. I did not test it with the sign up flow, but with updateUserAttribute, it was crashing at first because I had handled it the same way it was handled here.

Reply:
I will attempt to test this in the sign up flow to confirm its currently a bug, but @haverchuck I would appreciate your eyes on this one.

Reply:
Marking as resolved. After looking into this, the sign up flow doesn't allow users to pass in integer values. I updated the methods I added to match that.